torchbenchmark/util/backends/fx2trt.py

```diff
@@ -1,5 +1,6 @@
 import torch
 from typing import Tuple, Optional
+from fx2trt_oss.fx.lower import lower_to_trt
```

Comment:
Actually, please move this line in `enable_fx2trt()`, since our CI system only checks default settings and don't have fx2trt access